app/views/results/_location.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="filter-form" data-qa="filters__location">
+  <h2 class="govuk-heading-s filter-form__title">
+    Location<span class="govuk-visually-hidden">:</span>
+  </h2>
+  <ul class="filter-form__value--list">
+    <li data-qa="location_name">
+      <%= @results_view.location %>
+      <% if @results_view.distance %>
+        <span class="govuk-caption-m" data-qa="distance">Within
+          <%= @results_view.distance %>
+         miles of the pin</span>
+        <img src="https://maps.googleapis.com/maps/api/staticmap?key=<%= @results_view.maps_api_key %>&center=<%=
+            @results_view.latitude %>,<%= @results_view.longitude %>&zoom=<%=
+            @results_view.map_radius %>&size=300x200&scale=2&markers=<%=
+            @results_view.latitude %>,<%= @results_view.longitude %>"
+             alt="Map showing <%= @results_view.distance %> miles
+             of <%= @results_view.location %>" style="max-width: 100%"
```

Comment:
Should style not be in css file?

Reply:
yes

Reply:
@aliuk2012 could you help with this? I wouldn't know where to start with our css at the moment

Reply:
why inline style?

Reply:
I'd say as its part of `filter-form`  pattern you could add it to `webpacker/styles/patterns/_filters.scss`. 

Try adding this to the bottom of the `_filters.scss`

```css
.filter-form__map {
  width: 100%;
}
```

then add `class="filter-form__map"` to the img element

Reply:
thanks, done